docs/specs/servers/push/spec.md

```diff
@@ -80,27 +94,53 @@ DELETE <PUSH_SERVER_URL>/clients/<CLIENT_ID>
 
 ### Send Notification
 
+This endpoint is called by the relay.
+
 ```
 POST <PUSH_SERVER_URL>/clients/<CLIENT_ID>
+```
+
+- `CLIENT_ID` - This is used to map to the device token
+
+Headers:
+- `X-Ed25519-Timestamp` - timestamp of the signature
+- `X-Ed25519-Signature` - hex encoded ed25519 signature of `format!("{}.{}.{}", timestamp, body.len(), body)` where `body` is the request body
```

Comment:
@chris13524 why do we need this signature and which signature is it?

Reply:
This is the relay's signature that signs the webhook so that echo server knows the content is legitimate. This was already implemented, but undocumented, so I added it here.